projects/cdk/coercion/coerce-boolean-property.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Coerces a data-bound value (typically a string) to a boolean.
+ * @example {@link https://material.angular.io/cdk/coercion/api#functions}
+ */
+export function tuiCoerceBooleanProperty(
```

Comment:
please, make a simple unit test

Reply:
I took this util from 

https://github.com/angular/components/blob/bc2fddbc4190a63a6c59372e21238086beeec193/src/cdk/coercion/boolean-property.ts#L16-L18

Should I copy-paste its tests ?
https://github.com/angular/components/blob/bc2fddbc4190a63a6c59372e21238086beeec193/src/cdk/coercion/boolean-property.spec.ts#L3

Reply:
yes)

Reply:
✅